packages/indy-vdr/src/anoncreds/IndyVdrAnonCredsRegistry.ts

```diff
@@ -218,12 +219,14 @@ export class IndyVdrAnonCredsRegistry implements AnonCredsRegistry {
 
       const response = await pool.submitReadRequest(request)
 
-      if (response.result.data) {
+      const schema = await this.fetchIndySchemaWithSeqNo(agentContext, response.result.ref, did, 1)
```

Comment:
no need to pass type

```suggestion
      const schema = await this.fetchIndySchemaWithSeqNo(agentContext, response.result.ref, did)
```